src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs

```diff
@@ -48,6 +48,7 @@ public static class PropertyName
             public const string ExtensionMethods = "extensionMethods";
             public const string Overload = "overload";
             public const string Return = "return";
+            public const string SeeAlsoContent = "seealsoContent";
```

Comment:
want to be consistent with seealso

---
In reply to: [114701331](https://github.com/dotnet/docfx/pull/1625#discussion_r114701331) [](ancestors = 114701331)